tests/python/pants_test/backend/jvm/tasks/test_classpath_products.py

```diff
@@ -277,7 +276,7 @@ def test_jar_missing_pants_path_fails_adding(self):
                                                            cache_path='somewhere',
                                                            pants_path=None)])
     self.assertEqual(
-      'Jar: org:name: has no specified path.',
+      'Jar: org:name:::jar has no specified path.',
```

Comment:
This is a bit odd... I don't think the digest should be affecting the coordinate?